app/models/spree_avatax/config.rb

```diff
@@ -25,11 +27,25 @@ def enabled
         (config = active) ? config.enabled : true
       end
 
+      def customer_code
+        @customer_code || default_customer_code
+      end
+
+      def customer_code=(evaluator)
+        raise(Exception) unless evaluator.respond_to?(:call)
```

Comment:
I might use a more descriptive name for this exception, for readability's sake.

Reply:
Indeed, what do you think of avoiding an exception altogether by wrapping the argument _in_ a proc if it's not callable?